<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="windowFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>291</width>
      <height>114</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>114</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>114</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#windowFrame
{
    background-color: rgba(0, 0, 0, 200);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="headerFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>281</width>
       <height>108</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="playFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="backwardButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sliderFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="openButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="songSlider">
           <property name="styleSheet">
            <string notr="true">QSlider
{
	margin: 0px;
}
QSlider::groove:horizontal
{
	border-radius: 5px;
	height: 10px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover
{
	background-color: rgb(42, 49, 62);
}
QSlider::handle:horizontal
{
    border: none;
    height: 10px;
    width: 10px;
    margin: 0px;
    border-radius: 5px;
    background-color: rgb(189, 147, 249);
}
QSlider::handle:horizontal:hover
{
    background-color: rgb(199, 157, 255);
}
QSlider::handle:horizontal:pressed
{
    background-color: rgb(255, 121, 198);
}
</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="bodyFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>281</width>
       <height>0</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="songListFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>291</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QListWidget" name="songList"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
